ports/atmel-samd/boards/circuitplayground_express_displayio/mpconfigboard.mk

```diff
@@ -12,7 +12,9 @@ EXTERNAL_FLASH_DEVICES = "S25FL216K, GD25Q16C"
 
 # Turn off features and optimizations for Crickit build to make room for additional frozen libs.
 LONGINT_IMPL = NONE
+CIRCUITPY_AUDIOBUSIO = 0
```

Comment:
Is AUDIOBUSIO useful for this CPX build? How does it compare with the other CPX builds, in terms of features?

Reply:
oh -- yes it is -- PDMIn for the mic :microphone: .  Might have to find something else to cut.  I thought only in terms of I2SOut.